apps/experiments/tasks.py

```diff
@@ -54,7 +55,12 @@ def get_prompt_builder_response_task(team_id: int, user_id, data_dict: dict) ->
         last_user_message = input_formatter.format(input=last_user_message)
 
     # Get the response from the bot using the last message from the user and return it
-    answer, input_tokens, output_tokens = conversation.predict(last_user_message)
+    history = PromptBuilderHistory.objects.create(team_id=team_id, owner=user, history=[])
```

Comment:
would this give accurate values with compressed histories?

Reply:
The history compression happens before the LLM api is called so the token counts should be correct in terms of what get's sent to the LLM